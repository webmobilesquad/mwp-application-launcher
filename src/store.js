import createStore from 'unistore';

export const store = createStore({
  applications: {
    data: null,
    loading: false,
  },
});
