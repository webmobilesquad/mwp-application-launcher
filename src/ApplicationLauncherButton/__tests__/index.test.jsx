import render from 'preact-render-spy';
import { Provider } from 'unistore/preact';
import { ApplicationLauncherButton } from '../index';
import { store } from '../../store';

describe('<ApplicationLauncherButton />', () => {
  test('<ApplicationLauncherButton /> matches snapshot', () => {
    expect(
      render(
        <Provider store={store}>
          <ApplicationLauncherButton />
        </Provider>,
      ),
    ).toMatchSnapshot();
  });
});
