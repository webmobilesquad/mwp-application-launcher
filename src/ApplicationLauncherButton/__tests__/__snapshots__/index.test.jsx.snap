// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApplicationLauncherButton /> <ApplicationLauncherButton /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<div>
  <button
    type="button"
    class="ApplicationLauncherButton"
    onClick={[Function anonymous]}
  >
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
    >
      <path d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"></path>
      <path
        d="M0 0h24v24H0z"
        fill="none"
      >
      </path>
    </svg>
  </button>
  <aside class="ApplicationLauncherModal">
    <i grid={true}></i>
  </aside>
</div>

`;
